client/src/App.tsx

```diff
@@ -1,14 +1,26 @@
-import { Box } from "@mui/material";
-import Header from "./components/shared/header";
-import CompetePage from "./pages/compete";
+import { Route, Routes, BrowserRouter } from "react-router-dom";
+import HomePage from "./pages/Home";
+import CompetePage from "./pages/Compete";
+import { Grid } from '@mui/material';
+import Header from './components/Header';
 
 const App = () => {
   return (
-    <Box>
-      <Header />
-      <CompetePage />
-    </Box>
+    <Grid container>
+      <Grid item xs>
+        <Header />
+      </Grid>
+
+      <Grid container bgcolor={"background.default"}>
+        <BrowserRouter>
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="compete" element={<CompetePage />} />
```

Comment:
Isn't the path supposed to be with "/" in front of the "compete"?
```suggestion
            <Route path="/compete" element={<CompetePage />} />
```